lib/teams/emrContainersRbacConfig.ytpl

```diff
@@ -0,0 +1,41 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Let's rename the file to match our naming convention: `emr-containers-rbac-config.ytpl` (that is if we need this separate rbac config)